Parsers/ASimProcessEvent/Parsers/vimProcessCreateVMwareCarbonBlackCloud.yaml

```diff
@@ -0,0 +1,273 @@
+      let processdatawithoutthreat = processdata
+          | where isempty(alert_id_g) or isempty(event_id_g);
+      union processdatawithoutthreat, processdatawiththreat
+      | extend
+          EventStartTime = todatetime(split(createTime_s, '+')[0]),
+          TargetProcessId = tostring(toint(childproc_pid_d)),
+          ActingProcessId = tostring(toint(process_pid_d)),
+          ParentProcessId = tostring(toint(parent_pid_d)),
+          AdditionalFields_Common = bag_pack(
+                        "org_key", org_key_s,
+                        "alert_id", alert_id_g,
+                        "parent_cmdline", parent_cmdline_s,
+                        "process_reputation", process_reputation_s,
+                        "childproc_reputation", childproc_reputation_s,
+                        "parent_reputation", parent_reputation_s,
+                        "process_guid", process_guid_s,
+                        "childproc_guid", childproc_guid_s,
+                        "parent_guid", parent_guid_s
+                    )
+      | invoke _ASIM_ResolveDvcFQDN('device_name_s')
+      | project-rename 
+          ActorUsername = process_username_s,
+          TargetProcessCommandLine = target_cmdline_s,
+          TargetProcessName = childproc_name_s,
+          TargetUsername = childproc_username_s,
+          DvcIpAddr = device_external_ip_s,
+          DvcScope = device_group_s,
+          ActingProcessCommandLine = process_cmdline_s,
+          ActingProcessName = process_path_s,
+          DvcId = device_id_s,
+          DvcOriginalAction = sensor_action_s,
+          DvcOs = device_os_s,
+          EventMessage = event_description_s,
+          EventOriginalType = action_s,
+          EventOriginalUid = event_id_g,
+          EventOwner = event_origin_s,
+          ParentProcessName = parent_path_s,
+          EventUid = _ItemId
+      | extend
+          EventCount = int(1),
+          EventProduct = "Carbon Black Cloud",
+          EventSchemaVersion = "0.1.4",
+          EventType = "ProcessCreated",
+          EventVendor = "VMware",
+          EventSchema = "ProcessEvent",
+          AdditionalFields = bag_merge(AdditionalFields_threat, AdditionalFields_Common)
+      | extend 
+          Dvc = coalesce(DvcFQDN, DvcId, DvcHostname, DvcIpAddr),
+          EventEndTime = EventStartTime,
+          Hash = coalesce(TargetProcessSHA256, TargetProcessMD5),
+          CommandLine = TargetProcessCommandLine,
+          Process = TargetProcessName,
+          User = TargetUsername
+      | extend 
+          DvcIdType = iff(isnotempty(DvcId), "Other", ""),
+          ActorUsernameType = _ASIM_GetUsernameType(ActorUsername),
+          ActorUserType = _ASIM_GetUserType(ActorUsername, ""),
+          HashType = case(
+                isnotempty(TargetProcessSHA256),
+                "TargetProcessSHA256",
+                isnotempty(TargetProcessMD5),
+                "TargetProcessMD5",
+                ""
+            ),
```

Comment:
can we merge above multiple extend statements?

Reply:
Done